README.md

```diff
@@ -29,6 +29,37 @@ To do this, do the following:
 - From the `stencil-library` directory, run `yarn link`
 - From the `react-library` run `yarn link "@justifi/webcomponents"`
 
+## Lerna Management
+
+This project now uses [Lerna](https://lerna.js.org/) to manage the versions of the multiple packages. Here are some common Lerna commands you might need:
+
+### Bootstrapping
+
+Lerna will ensure dependencies are linked together:
+
+```bash
+lerna bootstrap
+```
+
+### Checking for Changes
+
+To see which packages have changed since the last release:
+
+```bash
+lerna changed
+```
+
+### Publishing Changes
```

Comment:
I am also not sure if we want to document this, because we should be doing very intentional versioning/publishing from a github action rather than running the commands from our machiens